src/lib/uicons.js

```diff
@@ -197,85 +74,84 @@ class Uicons {
 
 	async isUiconsRepository() {
 		const currentSet = await getAvailableIcons(this.log, this.url)
-
 		return !!currentSet
 	}
 
 	async pokemonIcon(pokemonId, form = 0, evolution = 0, gender = 0, costume = 0, alignment = 0, shiny = false) {
 		const currentSet = await getAvailableIcons(this.log, this.url)
-		if (currentSet) return `${this.url}/pokemon/${resolvePokemonIcon(currentSet.pokemon, this.imageType, pokemonId, form, evolution, gender, costume, alignment, shiny)}`
+		if (currentSet) return currentSet.pokemon(pokemonId, form, evolution, gender, costume, alignment, shiny)
 		if (this.fallback) return `${this.url}/pokemon_icon_${pokemonId.toString().padStart(3, '0')}_${form ? form.toString() : '00'}${evolution > 0 ? `_${evolution.toString()}` : ''}.${this.imageType}`
```

Comment:
doesn't work with: https://github.com/bbdoc/tgUICONS

> Failed to send Telegram sticker https://raw.githubusercontent.com/bbdoc/tgUICONS/main/Shuffle/pokemon_icon_339_00.webp

seems the fallback is triggered. the issue might be the file extension? `webp` instead of `png`?